dbt/tests/generic/test_column_is_subset_of_external_column.sql

```diff
@@ -0,0 +1,28 @@
+-- Test that a given column is a subset of a column in an external relation.
```

Comment:
I'm not totally sure how useful this test macro will be yet, but since the new test would have required a custom query script anyway, I figured I might as well start with the general case and see if it becomes useful.